### DISCUSSION TITLE
Is *.app.online.visualstudio.com the new preview URL?

### DISCUSSION THREAD

**Discussion by treeder** [community/community#20928] [Codespaces]
I noticed some of my codespaces have that URL suffix instead of *.githubpreview.dev. Which should we expect going forward?

**Comment by jfullstackdev** [comment]
I noticed that too, seems that's a new one 

update:

it's gone by now 🤣

**Reply by treeder** [reply]
Mine keeps flipping back and forth.

**Comment by Fengyi-Li** [comment]
githubpreview.dev domain is current latest domain, *.app.online.visualstudio.com is the previous one that should not be in use anymore. To help us better investigate the issue, can you please provide more details? 

Do you see the different URL when creating a codespace or restarting an existing one? Or does it flip back and forth when you are actively working on the codespace? Thank you!

**Comment by jkeech** [comment]
Apologies for this issue! The change is not expected. We've identified the problem and are working on rolling out a fix.

**Reply by jkeech** [reply]
The fix is rolled out. If you are still seeing this issue, please try the following:
1) Stop the codespace that you are connected to
2) Close your browser tab
3) Open a new tab and connect to your codespace
4) Forward a port and it should be using the format `https://{name}-{port}.githubpreview.dev`
